pom.xml

```diff
@@ -237,7 +237,8 @@
         <surefire.args>${surefire.args.base}</surefire.args>
 
         <commonsCollectionsVersion>3.2.2</commonsCollectionsVersion>
-        <groovyVersion>3.0.7</groovyVersion>
```

Comment:
Should we keep this one just in case it was overriding the BOM one? or have you verified that the one from the BOM (3.0.13) was always being used?